src/deadline/job_attachments/README.md

```diff
@@ -64,3 +64,23 @@ You can install the protocol handler by running the command: `deadline handle-we
 ## Security
 
 When creating a queue, provide the name of an S3 bucket in the same account and region as the queue you are creating, and provide a 'root prefix' name for files to be uploaded to. You also must provide an IAM role that has access to the S3 bucket. See the [security best practices][ja-security] documentation for more information on securely configuring job attachments.
+
+## Job Attachments Subcommands
+
+Job Attachments provides `attachment` and `manifest` subcommand groups as part of deadline command-line interface.
+
+Attachment subcommands work with data files based on asset manifest files.
+
+```sh
+$ deadline attachment upload
```

Comment:
Since we are "officially" adding this to readme, do we need to add any explanation of what the commands do here?

Reply:
I didn't add them here since they're in the commands and visible via `-h`. Other commands don't have explanation in README either. Keeping them in both places can easily go out of date.

Reply:
ok - The examples are probably much more useful.